libs/ui/src/accessible_controllers/test_utils.tsx

```diff
@@ -1,5 +1,5 @@
 import userEvent from '@testing-library/user-event';
 
 export function simulateKeyPress(key: string): void {
-  userEvent.keyboard(key.length === 1 ? key : `[${key}]`);
+  userEvent.keyboard(key.length === 1 ? key : `{${key}}`);
```

Comment:
The square brace syntax didn't seem to work for the `Shift` key, so had to switch to the curly brace syntax (https://testing-library.com/docs/user-event/keyboard/)